### PR TITLE
chore(Info): Info pools redirection

### PR DESCRIPTION
**Pull request by Woody810**

### CODE REVIEW

File: apps/web/next.config.mjs [hunk 1]
```diff
@@ -150,6 +158,11 @@ const config = {
         destination: '/nfts',
         permanent: true,
       },
+      {
+        source: '/info/pairs',
+        destination: '/info/pools',
```
Comment (chef-jojo) [inline apps/web/next.config.mjs:163]: Should invert?
old pools to pairs

Comment (Woody810) [reply apps/web/next.config.mjs:163]: let me check

Comment (Woody810) [reply apps/web/next.config.mjs]: fixed

File: apps/web/next.config.mjs [hunk 2]
```diff
@@ -75,6 +75,14 @@ const config = {
         source: '/info/pair/:address',
         destination: '/info/pools/:address',
```
Comment (chef-jojo) [inline apps/web/next.config.mjs:76]: pairs

File: apps/web/next.config.mjs [hunk 3]
```diff
@@ -75,6 +75,14 @@ const config = {
...
       },
+      {
+        source: '/info/pairs',
+        destination: '/info/pools',
+      },
+      {
+        source: '/info/pairs/:address',
+        destination: '/info/pools/:address',
+      },
```
Comment (chef-jojo) [inline apps/web/next.config.mjs:85]: No need rewrites those

**Review by chef-jojo — CHANGES_REQUESTED** [review]

File: apps/web/next.config.mjs (hunk 2 above)
Comment (Woody810) [reply apps/web/next.config.mjs:76]: fixed

File: apps/web/next.config.mjs (hunk 3 above)
Comment (Woody810) [reply apps/web/next.config.mjs]: fixed

File: apps/web/next.config.mjs [hunk 4]
```diff
@@ -72,9 +72,13 @@ const config = {
...
       },
       {
-        source: '/info/pair/:address',
+        source: '/info/pairs/:address',
         destination: '/info/pools/:address',
       },
+      {
+        source: '/info/pairs',
+        destination: '/info/pools',
+      },
```
Comment (chef-jojo) [inline apps/web/next.config.mjs:81]: remove

File: apps/web/next.config.mjs (hunk 2 above)
Comment (chef-jojo) [reply apps/web/next.config.mjs:76]: nah should be pair -> pairs

File: apps/web/next.config.mjs (hunk 4 above)
Comment (Woody810) [reply apps/web/next.config.mjs]: done

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**